docs/Key-Events.md

```diff
@@ -0,0 +1,91 @@
+# Table/List of key modifications of LightGBM
+
+The list includes the commits where the major feature added is considered working with the least amount of flaws. This is useful if you are trying to get a specific commit, such as the first properly working commit for categorical support.
+
+## Modification Table
+
+| Date | Commit | Type | Description |
+| --- | --- | --- | --- |
+| 18/04/2017 | [7339ed6](https://github.com/Microsoft/LightGBM/pull/426) | Feature | Whitespaces Removed from Features support (Pull Request 426)
+| 14/04/2017 | [9224a9d](https://github.com/Microsoft/LightGBM/pull/415) | Release | GPU support (Pull Request 415)<br>With original ([0bb4a82](https://github.com/Microsoft/LightGBM/pull/368) on 09/04/2017, Pull Request 368)
+| 13/04/2017 | [ab55910](https://github.com/Microsoft/LightGBM/pull/404) | Feature | Speed Improvements for Prediction (Pull Request 404)
+| 06/04/2017 | [bfb0217](https://github.com/Microsoft/LightGBM/pull/383) | Feature | Objective Transformations to the Objective (Pull Request 383)
+| 05/04/2017 | [d4c4d9a](https://github.com/Microsoft/LightGBM/pull/381) | Feature | Regression Speed Improvements (Pull Request 381)
+| 03/04/2017 | [b6c973a](https://github.com/Microsoft/LightGBM/pull/378) | Feature | Unloading Memory Fix support for R package (Pull Request 378)
+| 17/03/2017 | [06a915a](https://github.com/Microsoft/LightGBM/pull/347) | Feature | RDS support for R pacakge (Pull Request 347)
+| 10/03/2017 | [b7e5f07](https://github.com/Microsoft/LightGBM/pull/340) | Feature | Support Self-Contained R package (Pull Request 340)
+| 02/03/2017 | [4d6ff28](https://github.com/Microsoft/LightGBM/pull/330) | Feature | Feature Contribution Plot for R package (Pull Request 330)
+| 01/03/2017 | [166421e](https://github.com/Microsoft/LightGBM/pull/328) | Feature | Feature Importance Plot for R package (Pull Request 328)
+| 01/03/2017 | [083b5da](https://github.com/Microsoft/LightGBM/pull/314) | Parameter | Unbalanced Mutliclass support (Pull Request 314)
```

Comment:
removed.

Do you have any important commits (not from PRs) you consider key events that should be added?